config/Makefile

```diff
@@ -18,5 +18,5 @@ SUFFIXES:
 include ../Makefile.INCLUDE
 
 generated/config.pb.go: config.proto
-	$(GO_GET) code.google.com/p/goprotobuf/protoc-gen-go
+	$(GO) get github.com/golang/protobuf/protoc-gen-go
```

Comment:
This is a compromise. If we use `$(GO_GET)`, it will try to put it into the existing protobuf directory copied over from the Godep vendored files, but that doesn't include the `.hg` directory anymore, so it fails.